README.md

```diff
@@ -72,6 +74,9 @@ setup:
 		echo AWS_PAGER= >> $(TEMP_ENV_FILE)
 	fi
 
+	echo "RG_LOCATION_SHORT=$(RG_LOCATION_SHORT)" >> $(TEMP_ENV_FILE)
+	echo "RG_LOCATION_LONG=$(RG_LOCATION_LONG)" >> $(TEMP_ENV_FILE)
```

Comment:
Looking at the other configurations, we don't enclose the text in `"` on the other places.